src/Files/Dialogs/FilesystemOperationDialog.xaml.cs

```diff
@@ -85,6 +86,14 @@ private void Grid_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
                 (sender as Grid).FindAscendant<ListViewItem>().ContextFlyout = ItemContextFlyout;
             }
         }
+
+        private void FilesystemOperationDialog_OnLoaded(object sender, RoutedEventArgs e)
+        {
+            chkPermanentlyDelete.IsEnabled = ViewModel.PermanentlyDeleteEnabled;
```

Comment:
PermanentlyDelete was a two-way binding, you may need to register to the Checked event and update the viewmodel.

Reply:
I’ll make that change shortly. Thanks.

Reply:
(I haven't tested this but) do all "x:Bind"s cause the focus issue? Or could it be that it's only the "ItemsSource" binding for example that causes the issue.

Reply:
It definitely seemed to be all the x:Binds from my testing. I might just double check that though when I make these changes. 

I don’t really understand why it’s causing an issue. I tried setting the default button after the dialog had loaded but even that didn’t work.

Reply:
Okay, apparently my previous commit "broke" the fix somehow. I think it has something to do with the order that the PrimaryButton is enabled and the ListView. The Checkbox with the x:Binds seems to not have any affect. I have just double checked that it is working as intended with how it is now. 

I may come back to this later try work through it again when I'm not so tired.